src/router.ts

```diff
@@ -18,16 +37,14 @@ export function createRouter(withRouter: (router: Router) => Router = identity):
 
   const router = new Router()
     .get(`/health-check`, ({ response }) => Object.assign(response, { status: 200, body: 'OK' }))
-    .get('/', ctx => send(ctx, '/public/index.html'))
-    .get('/welcome', ctx => send(ctx, '/public/welcome.html'))
+    .get('/', ctx => ctx.render('index'))
+    .get('/welcome', ctx => ctx.render('welcome'))
     .use('/v1', v1Router.routes(), v1Router.allowedMethods())
 
-  // tslint:disable-next-line:no-expression-statement
-  readdirSync(process.cwd() + '/public').forEach(fileName => {
-    const file = parse(fileName)
-    const route = file.ext === '.html' ? file.name : fileName
-    return router.get('/' + route, ctx => send(ctx, `/public/${fileName}`))
-  })
+  for (const file of files('/public')) {
+    const route = file.ext === '.html' ? file.name : file.base
+    router.get('/' + route, ctx => send(ctx, file.full)) // tslint:disable-line:no-expression-statement
```

Comment:
Can't we use `ctx.render` here as well?

Reply:
These are mostly for the css and favicon files. The auth-success page is weird, but I think we can leave it in for now